.babelrc.cjs

```diff
@@ -11,9 +11,7 @@ module.exports = function (api) {
         },
       ],
     ],
-    plugins: [
-      ["module-extension", { js: "cjs" }], // The built files need to use the .cjs extension to import other relative files
```

Comment:
This plugin didn't have an option to ignore non-relative imports, so we swapped it with a simple custom one.